frontend/src/components/SlidebarSettingPage.tsx

```diff
@@ -0,0 +1,79 @@
+                            <div style={{content:'" "', height:'8px', background:'#192A51', bottom:'-2px', left:'20px'}} className="mt-1 ms-1.5 sm:ms-0 px-10  rounded-full" ></div> )}
+                            { (hoverGraphs && showDiv === false)&& (
+                            <div style={{content:'" "', height:'8px', background:'#192A51', bottom:'-2px', left:'20px'}} className="mt-1 ms-0 sm:ms-0 px-10  rounded-full" ></div> )}
+                        </div>
+                        <div onMouseEnter={() => setHoverPowers(true)}
+                             onMouseLeave={() => setHoverPowers(false)}>
+                            <a href="powers"   className="" > Powers  </a>
+                            { (pathValue === 'powers' && showDiv) && (
+                            <div style={{content:'" "', height:'8px', background:'#192A51', bottom:'-2px', left:'20px'}} className="mt-1 ms-1.5 sm:ms-0 px-10 rounded-full" ></div> )}
+                            { (hoverPowers && showDiv === false)&& (
+                            <div style={{content:'" "', height:'8px', background:'#192A51', bottom:'-2px', left:'20px'}} className="mt-1 ms-0 sm:ms-0 px-10  rounded-full" ></div> )}
+                        </div>
+                        <div onMouseEnter={() => setHoverNetworking(true)}
+                             onMouseLeave={() => setHoverNetworking(false)}>
+                            <a href="networking"  > Networking  </a>
+                            { (pathValue === 'networking' && showDiv) && (
+                            <div style={{content:'" "', height:'8px', background:'#192A51', bottom:'-2px', left:'20px'}} className="mt-1 ms-1.5 sm:ms-0 px-10 rounded-full "></div>)} 
+                            { (hoverNetworking && showDiv === false) && (
+                            <div style={{content:'" "', height:'8px', background:'#192A51', bottom:'-2px', left:'20px'}} className="mt-1 ms-0 sm:ms-0 px-10  rounded-full" ></div> )}
+                        </div>
+                        <div onMouseEnter={() => setHoverDestroy(true)}
+                             onMouseLeave={() => setHoverDestroy(false)}>
+                            <a href="destroy"  > Destroy  </a>
+                            { (pathValue === 'destroy' && showDiv) && (
+                            <div style={{content:'" "', height:'8px', background:'#192A51', bottom:'-2px', left:'20px'}} className="mt-1 ms-1.5 sm:ms-0  px-10   rounded-full" ></div>)}
+                            { (hoverDestroy && showDiv === false)&& (
+                            <div style={{content:'" "', height:'8px', background:'#192A51', bottom:'-2px', left:'20px'}} className="mt-1 ms-0 sm:ms-0 px-10  rounded-full" ></div> )}
+                        </div>
+                     
+               </div>
+               <div className="absolute shadow-md top-36 ml-2 mt-2 px-[50%] py-1 bg-red-300 bg-opacity-50 z-[-1] rounded-full"></div>
```

Comment:
1. don't use style={}, if want to have logic in tailwind suggest use https://www.npmjs.com/package/tailwind-merge and use logic in their instead 
2. could we convert into reusable component? 
<NavLink isCurrentPage={true} >link text</NavLink>